examples/mixed/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "splunk-otel-example-mixed",
+  "private": true,
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "start": "node -r @splunk/otel/instrument index.js"
+  },
+  "dependencies": {
+    "@opentelemetry/api": "^1.0.2",
+    "@opentelemetry/instrumentation-http": "^0.23.0",
```

Comment:
Why not use latest 0.25.0? @splunk/otel 0.12.0 seems to use  0.25

Reply:
See: `npm info @splunk/otel@0.12 peerDependencies`
It's because later versions would conflict with our peerDeps. Older npm's don't care, but v7 will. That was actually one of the bugs I caught using the tests on our examples!